tests/test_integration.py

```diff
@@ -0,0 +1,52 @@
+# SPDX-License-Identifier: MIT
+
+import os
+import os.path
+import re
+
+import pytest
+
+import build.__main__
+
+
+_SDIST = re.compile('.*.tar.gz')
+_WHEEL = re.compile('.*.whl')
+
+
+@pytest.mark.parametrize(
+    ('project'),
+    [
+        'python-build',
+        'pip',
+        'dateutil',
+        'Solaar',
+    ]
+)
+@pytest.mark.parametrize(
+    ('args'),
+    [
+        [],
+        ['-x', '--no-isolation'],
+    ]
+)
+def test_build(tmpdir, integration_path, project, args):
+    os.environ['SETUPTOOLS_SCM_PRETEND_VERSION'] = 'dummy'  # for the projects that use setuptools_scm
+
+    build.__main__.main([os.path.join(integration_path, project), '-o', tmpdir] + args)
+
+    assert filter(_SDIST.match, os.listdir(tmpdir))
+    assert filter(_WHEEL.match, os.listdir(tmpdir))
+
+
+def test_isolation(tmpdir, test_flit_path, mocker):
+    try:
+        import flit_core  # noqa: F401
+    except:  # noqa: E722
+        pass
+    else:
+        pytest.xfail('flit_core is available')
```

Comment:
simpler:

```python
with pytest.raises(ImportError):
    import flit_core
```

Reply:
I want to skip the test if `flit_core` is available, not fail.